src/Components/Dashboard/Dashboard.tsx

```diff
@@ -1,7 +1,157 @@
+      .get(`${baseUrl}/Task/count`, { headers: requestHeaders })
+      .then((response) => {
+        setIsLoading(false);
+        // console.log(response?.data);
+        setTodoCount(response?.data?.toDo);
+        setProgressCount(response?.data?.inProgress);
+        setDoneCount(response?.data?.done);
+      })
+      .catch((error) => {
+        setIsLoading(false);
+        // console.log(error);
+      });
+  };
+  const getUsersCount = async () => {
+    setIsLoading(true);
+    await axios
+      .get(`${baseUrl}/Users/count`, { headers: requestHeaders })
+      .then((response) => {
+        setIsLoading(false);
+        // console.log(response.data);
+        setActiveUserCount(response?.data?.activatedEmployeeCount);
+        setInActiveUserCount(response?.data?.deactivatedEmployeeCount);
+      })
+      .catch((error) => {
+        setIsLoading(false);
+        // console.log(error);
+      });
+  };
+
+  useEffect(() => {
+    getTasksCount();
+    getUsersCount();
+  }, []);
 
-export default function Dashboard() {
   return (
-    <div>Dashboard</div>
-  )
-}
+    <>
+      <div className="bg-header m-5">
+        <div className="bg-overlay">
+          <div className="p-5 text-white">
+            <h3 className="">
+              Welcome <span className="title">{/* {userData.userName} */}</span>
+            </h3>
+            <p className="w-75 ">
+              You can add project and assign tasks to your team
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className="container">
+        <div className="row">
+          <div className="col-md-6">
+            <div className="titles">
+              <h3>Tasks</h3>
+              <p>List of all tasks</p>
+            </div>
+            <div className="row tasks-count mt-5">
+              <div className="col-md-4 ">
+                <div className="tasks-status d-flex flex-column text-center p-3  rounded-4">
+                  <i className="fa fa-tasks my-3"> </i>
+                  <span className="py-3">To do</span>
+                  <h3>{todoCount}</h3>
+                </div>
+              </div>
+              <div className="col-md-4 ">
+                <div className="tasks-status d-flex flex-column text-center p-3 rounded-4">
+                  <i className="fa-solid fa-hourglass-end my-3"></i>
+                  <span className="py-3">In-progress</span>
+                  <h3>{progressCount}</h3>
+                </div>
+              </div>
+              <div className="col-md-4">
+                <div className="tasks-status d-flex flex-column text-center  p-3  rounded-4">
+                  <i className="fa-solid fa-clipboard-check my-3"></i>
+                  <span className="py-3">Done</span>
+                  <h3>{doneCount}</h3>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="col-md-6">
+            <div className="titles">
+              <h3>Users</h3>
```

Comment:
Users Charts and boxes should be removed from employee dashboard.
You can render them conditionally based on the the userRole.

Reply:
done